hellojs/hellojs.d.ts

```diff
@@ -27,8 +27,14 @@ interface HelloJSEvent {
     success(callback: (json?: any) => void): HelloJSStatic;
     error(callback: (json?: any) => void): HelloJSStatic;
     complete(callback: (json?: any) => void): HelloJSStatic;
+    then(successCallback: (json?: any) => void, errorCallback: (json?: any) => void): HelloJSStatic;
 }
 
+interface HelloJSThenable {
+    then(successCallback: (json?: any) => void, errorCallback: (json?: any) => void): HelloJSStatic;
```

Comment:
Promises/A+ `.then` must always return a `then`able.

Also, perhaps `HelloJSEvent` should `extends HelloJSThenable`?